examples/config/_common/common.runtime.properties

```diff
@@ -16,16 +16,18 @@
 #
 
 # Extensions (no deep storage model is listed - using local fs for deep storage - not recommended for production)
-druid.extensions.coordinates=["io.druid.extensions:druid-examples","io.druid.extensions:druid-kafka-eight","io.druid.extensions:mysql-metadata-storage"]
+# Also, for production to use mysql add, "io.druid.extensions:mysql-metadata-storage"
+druid.extensions.coordinates=["io.druid.extensions:druid-examples","io.druid.extensions:druid-kafka-eight"]
 
 # Zookeeper
 druid.zk.service.host=localhost
 
-# Metadata Storage (mysql)
-druid.metadata.storage.type=mysql
-druid.metadata.storage.connector.connectURI=jdbc\:mysql\://localhost\:3306/druid
-druid.metadata.storage.connector.user=druid
-druid.metadata.storage.connector.password=diurd
+# Metadata Storage (use something like mysql in production by uncommenting properties below)
+# by default druid will use derby
+# druid.metadata.storage.type=mysql
+# druid.metadata.storage.connector.connectURI=jdbc\:mysql\://localhost\:3306/druid
```

Comment:
while we're editing this, I don't think the escaping `:` is necessary, so we should remove it. `druid.metadata.storage.connector.connectURI=jdbc:mysql://druid.myhost.com/druid?characterEncoding=UTF-8` works perfectly fine for us